src/backend/utils.ts

```diff
@@ -1165,6 +1166,19 @@ function removeFolder(path: string, folderName: string) {
   return
 }
 
+function sendGameStatusUpdate(payload: unknown) {
+  sendFrontendMessage('gameStatusUpdate', payload)
+  backendEvents.emit('gameStatusUpdate', payload)
+}
+
+function sendProgressUpdate(payload: {
+  appName: string
+  [key: string]: unknown
```

Comment:
one more suggestion, now that we have these specific methods I think we could add a type for the `payload` to match what we already use it for